src/test/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensionSpec.scala

```diff
@@ -387,6 +388,7 @@ class GoogleExtensionSpec(_system: ActorSystem) extends TestKit(_system) with Fl
 
     val petServiceAccount2 = googleExtensions.createUserPetServiceAccount(defaultUser, googleProject).unsafeRunSync()
     petServiceAccount.serviceAccount shouldNot equal(petServiceAccount2.serviceAccount)
+    regDAO.loadPetServiceAccount(petServiceAccount.id).unsafeRunSync() shouldBe Some(petServiceAccount2)
```

Comment:
Is this test failing without your changes?

Reply:
The test as it was before was succeeding in dev sam. I added this assertion to the test before I made the fix in `GoogleExtensions` and saw that the assertion was failing because the `RegistrationDAO` (LDAP) had the wrong value for the pet service account's google subject id. After I made the fix, this test passed again.